scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -156,6 +156,13 @@ def _expand_path(path: Path) -> List[Path]:
         return [path]
 
 
+class ComponentType(Enum):
+    """Enum for component types."""
+
+    PIPELINE = "component"  # A pipeline component which allows multi-stage jobs.

```

Comment:
shouldn't this be PIPELINE = "pipeline"?

Reply:
yes

Reply:
Also I'm assuming the enum is being added just for future-proofing, since the other values aren't being used right now?

Reply:
yeap!